### ISSUE TITLE
[CITS_Geospatial_v3.pdf] Add knowledge graphs

### ISSUE THREAD

**Issue by anitagraser** [DILCISBoard/CITS-Geospatial#12]
p.16 

To support automatic knowledge discovery / GeoAI / geographic question answering, the metadata section may be extended to include knowledge graphs (e.g. in RDF representation).

The behaviour section should then include relevant (Geo)SPARQL or Cypher queries. 

Fore more details on geospatial knowledge graphs, see https://github.com/DILCISBoard/CITS-Geospatial/wiki/Spatial-knowledge-graphs

![image](https://github.com/DILCISBoard/CITS-Geospatial/assets/590385/e7bcd359-e534-4dd1-a122-babe5cfc0f0d)

**Comment by GregorZavrsnik** [comment]
This is a valid point. We will add additional information to the Guidelines for CITS Geospatial referring to these new topics in a way that is aligned with the common specifications approach to knowledge graphs and Linked data.

**Comment by anitagraser** [comment]
RDF triples can simply be archived as CSV

Challenges for archiving knowledge graphs include:

- Many thesauri are not stable enough for long-term archiving 
- Graph DB exports, e.g. from the popular Neo4J database contain more information than can be represented in RFD triples. A RDF compatibility plugin for Neo4J does exist, however, it remains to be determined whether / to what degree this can be helpful for archiving

**Comment by anitagraser** [comment]
Ad behavior: 

In addition to [SPARQL](https://en.wikipedia.org/wiki/SPARQL) (a W3C standard declarative query language for querying graph data) there is also [Cypher](https://en.wikipedia.org/wiki/Cypher_(query_language)) for Neo4j which was opened up through the [openCypher](https://opencypher.org/) project in October 2015.

Spatial support is available via GeoSPARQL and [Cypher spatial functions](https://neo4j.com/docs/cypher-manual/current/functions/spatial/), respectively.

**Comment by GregorZavrsnik** [comment]
The CITS Geospatial is written in such a general way, that we already have the placeholders for certain elements of the Knowledge graphs and linked data. Therefore we added several mentions in the Guideline for CITS Geospatial on this topic:
- A whole new subchapter 2.1.5 Geospatial records in linked data environment
- GEO_11 (minimum 1 geospatial file in the package) has additional text describing cases where we are referring external distributed sources, which applies to linked data as well
- to GEO_27 and GEO_29 (Standardised machine-readable Feature Catalogue and logical model) a paragraph was added describing possibilities of storing taxonomies ontologies and or vocabularies in standardized formats
- GEO_39 (Common queries and algorithms) GeoSPARQL was also added as a possible example. 
- 3.5. Geospatial metadata requirements - DCAT and GeoDCAT was mentioned as a possible metadata format.

There is more to do on this topic, which we will aim to deliver in future releases.
